src/OrchardCore.Modules/OrchardCore.Markdown/Views/MarkdownField.Edit.cshtml

```diff
@@ -7,10 +7,9 @@
 @{
     var settings = Model.PartFieldDefinition.GetSettings<MarkdownFieldSettings>();
     var culture = await Orchard.GetContentCultureAsync(Model.Field.ContentItem);
-    var uniqueName = $"{Model.PartFieldDefinition.PartDefinition.Name}-{Model.PartFieldDefinition.Name}";
 }
 
-<div class="@Orchard.GetWrapperCssClasses("field-wrapper", $"field-wrapper-{uniqueName.HtmlClassify()}")" id="@Html.IdFor(x => x.Markdown)_FieldWrapper">
+<div class="@Orchard.GetFieldWrapperCssClasses(Model.PartFieldDefinition)" id="@Html.IdFor(x => x.Markdown)_FieldWrapper">
```

Comment:
please change the whole block of HTML to this

```
<div class="@Orchard.GetFieldWrapperCssClasses(Model.PartFieldDefinition)" id="@Html.IdFor(x => x.Markdown)_FieldWrapper">
    @await DisplayAsync(await New.ShortcodeModal())
    <label asp-for="Markdown" class="@Orchard.GetLabelCssClasses()">@Model.PartFieldDefinition.DisplayName()</label>
    <div class="@Orchard.GetEndCssClasses()">
        <textarea asp-for="Markdown" rows="5" class="form-control content-preview-text shortcode-modal-input" dir="@culture.GetLanguageDirection()"></textarea>
        <span class="hint">@settings.Hint</span>
    </div>
</div>
```

Reply:
@MikeAlhayek why no if condition?
```
        @if (!string.IsNullOrEmpty(settings.Hint))
        {
            <span class="hint">@settings.Hint</span>
        }
```

Reply:
@lampersky 
just missed it. Please add the condition.

Reply:
@MikeAlhayek are you sure? Previously in **MarkdownField.Edit.cshtml** we had **input**, not **textarea**. I think you copy-pasted wrong code.

I will use this instead:

```
<div class="@Orchard.GetFieldWrapperCssClasses(Model.PartFieldDefinition)" id="@Html.IdFor(x => x.Markdown)_FieldWrapper">
    @await DisplayAsync(await New.ShortcodeModal())
    <label asp-for="Markdown" class="@Orchard.GetLabelCssClasses()">@Model.PartFieldDefinition.DisplayName()</label>
    <div class="@Orchard.GetEndCssClasses()">
        <input asp-for="Markdown" class="form-control content-preview-text shortcode-modal-input" dir="@culture.GetLanguageDirection()" />
        @if (!string.IsNullOrEmpty(settings.Hint))
        {
            <span class="hint">@settings.Hint</span>
        }
    </div>
</div>
```